rolecall/rolecall.py

```diff
@@ -135,28 +146,34 @@ def _check_entry(self, entry: Entry):
         server = ctx.message.server
         author = ctx.message.author
 
-        # retrieve role mentions in the command message
-        roles = ctx.message.raw_role_mentions
-        if len(roles) == 1:
-            role_object = discord.utils.get(server.roles, id=roles[0])
+        # check if role was mentioned 
+        nums_found = re.findall('\d+', role)
+        if nums_found:
+            potential_role_id = nums_found[0]
+            if potential_role_id in [r.id for r in server.roles]:
+                role_object = discord.utils.get(server.roles, id=potential_role_id)
         else:
             role_object = await self.get_or_create("role", role, server)
-
-        # retrieve channel mentions in the command message
-        channels = ctx.message.raw_channel_mentions
-        if len(channels) == 2:
-            role_channel = self.bot.get_channel(channels[1])
+            
+        # check if channel was mentioned
+        if private_channel is None: 
+            pass
         else:
```

Comment:
can we do `if private_channel is not None: do_things()` instead?

Reply:
I think I prefer it this way instead of using double negatives that may be mildly confusing to some people, or to me when I'm sleepy :3